spec/io_spec.rb

```diff
@@ -26,7 +26,7 @@
 #
 require "tmpdir" # Used to avoid cluttering the repository.
 require_relative 'spec_helper'
-require_relative "../lib/nmatrix"
+require "./lib/nmatrix"
```

Comment:
Keep Travis as a future task? No. This is required for merging any new code. The tests that were failing for MRI were succeeding previously; either something changed with Travis or something was broken when you made your JRuby-related changes.